dali/server/daserver.cpp

```diff
@@ -139,6 +139,58 @@ void usage(void)
     printf("--daemon|-d <instanceName>\t: run daemon as instance\n");
 }
 
+static IPropertyTree *secMgrPluginPropTree(const IPropertyTree *configTree)
+{
+    Owned<IPropertyTree>    foundTree;
+
+#ifdef _CONTAINERIZED
+    IPropertyTree *jwtConfigPTree = configTree->queryPropTree("jwt");
+
+    if (jwtConfigPTree)
+    {
+        foundTree.set(createPTreeFromXMLString("<SecurityManagers/>"));
+
+        StringAttr configName(jwtConfigPTree->queryProp("@objname"));
+
+        VStringBuffer secMgrXML("<SecurityManager name='%s' instanceFactoryName='createInstance' libName='libjwtSecurity.so' type='JwtSecurityManager'/>", configName.str());
```

Comment:
This is admittedly terrible, and I blame Tony.  This code is inspired by the changes to esp/platform/application_config.cpp where, as a way to bridge the container-aware and legacy configurations, we basically replicate the the legacy XML configuration structure when running in a container.  The alternative to coding names like this would be to include them in YAML.

@afishbeck Would you recommend moving the library name here (and the names in application_config.cpp) into the YAML configurations or leave them coded?

Reply:
As part of another feature I'm adding a generic way of adding SecurityManager plugins.  If you want to wait for that for containerized mode that would be fine.  The new stuff does have the library as part of the yaml.

Reply:
Excellent, thank you.  I'll leave this as-is for now, pending your changes.